app/models/history.rb

```diff
@@ -66,9 +66,9 @@ class History < ApplicationRecord
     when 'Last 24 Hours'
       where('histories.created_at >= ?', 24.hours.ago)
     when 'Last 7 Days'
-      where('histories.created_at >= ? AND histories.created_at < ?', 7.days.ago.to_date.to_datetime, Date.today.to_datetime)
+      where('histories.created_at >= ? AND histories.created_at < ?', (DateTime.now.utc - 7.days).to_date.to_datetime, DateTime.now.utc.to_date.to_datetime)
```

Comment:
It seems like `.to_date.to_datetime` is being as a proxy for `DateTime#beginning_of_day`

Reply:
updated to use `beginning_of_day`